mteb/tasks/Clustering/dan/WikiClusteringP2P.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class WikiClusteringP2P(AbsTaskClustering):
+    metadata = TaskMetadata(
+        name="WikiClusteringP2P",
+        description="Clustering of wikipedia articles inspired by BlubrbsClusteringP2P. Labels are taken from top-level categories as defined in https://da.wikipedia.org/wiki/Kategori:Topniveau_for_emner.",
+        reference=None,
+        dataset={
+            "path": "ryzzlestrizzle/dawiki-clustering-p2p",
+            "revision": "4c4c6b6ad63311d6a072734d86dca1c0b48c441c",
+        },
+        type="Clustering",
+        category="p2p",
+        eval_splits=["test"],
+        eval_langs=["dan-Latn"],
+        main_score="v_measure",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples={"test": 150_000},
```

Comment:
downsample to say 2048 pr. language? Should also help you run the models.

Also makes the benchmark faster to run, which we would love.

Reply:
That might be too little as the 150k comes from 30 splits x 5k sentences. But 500-1k per split might be better?

Reply:
There are 20-40 categories depending on language. So maybe we could do 10 x 256?

Reply:
I would do it using relatively large splits: 5 x 2048 and then see if reducing it dramatically changes performance.

Reply:
I have now run it with 30 x 500 and the results are definitely reasonable. I'll try to resample to the suggested and update.

Reply:
Based on the results, I think I will update to 10 x 1024 and rerun - then I think it's pretty good:))